python/ray/experimental/placement_group.py

```diff
@@ -13,11 +13,12 @@ def placement_group(bundles: List[Dict[str, float]],
     Args:
         bundles: A list of bundles which represent the resources needed.
         strategy: The strategy to create the placement group.
-            There are two build-in strategies for the time begin.
+            There are three build-in strategies for the time begin.
```

Comment:
Can we just remove numbers? It looks cumbersome to update this whenever we add new strategies.

Reply:
fixed